src/datasets/other/attitudes_daily/intro.md

```diff
@@ -0,0 +1,15 @@
+The `attitudes_daily` table is a Telemetry instantiation of user responses to the [Daily Attitudes Survey (DAS)](https://qsurvey.mozilla.com/collab/daily-attitude-survey) over time. 
+It is joined to `clients_daily` using `client_id` and `submission_date`.
+
+#### Contents
+
+Most Firefox surveys are point-in-time without longitudinal insights.
+The DAS is completed by ~300 Firefox users every day, allowing us to measure long term attitudinal trends combined with users' corresponding Telemetry attributes.
+
+#### Accessing the Data
+
+The `attitudes_daily` table is accessible through re:dash using the
+`Telemetry (BigQuery)` data source.
+The full table name is `moz-fx-data-shared-prod.telemetry_derived.attitudes_daily_v1`.
```

Comment:
I just submitted mozilla/bigquery-etl#652 which creates a user-facing view for the table accessible as `telemetry.attitudes_daily` in STMO – in general we should prefer that view so we can make changes at the view level as necessary

Reply:
👍